test/utils/wabt_engine.cpp

```diff
@@ -31,15 +31,20 @@ std::unique_ptr<WasmEngine> create_wabt_engine()
 
 bool WabtEngine::parse(bytes_view input)
 {
+    wabt::interp::Environment env;
+    wabt::interp::DefinedModule* module{nullptr};
```

Comment:
Whose's the owner of this pointer? Is it a smart pointer?

Reply:
This rather works as out param and env owns it.

Reply:
Ok.